style.css

```diff
@@ -0,0 +1,105 @@
+* {
+    box-sizing: border-box;
+}
+a {
+    color: inherit;
+    text-decoration: inherit;
+}
+body {
+    margin: 0;
+    font-size: 20px;
+    font-family: 'Josefin Sans', sans-serif;
+    color: #aca9a5;
```

Comment:
apparently not doing anything

Reply:
but actually might somewhere, can't see it though.

Reply:
otherwise, nav text disappears once scrolled past hero image but if another nav function is going in then no issue obvs.

Reply:
I'm just setting precedents for the rest of page so I can use inherit functions and rems if necessary.